### PR TITLE
Couple fewer facets

### PR DESCRIPTION
**Pull request by emord**
@snopoke slightly fewer uses of facets. admin reports looks like it could be a bit nasty to get rid of

### CODE REVIEW

File: custom/_legacy/pact/utils.py [hunk 1]
```diff
@@ -66,30 +66,35 @@ def query_per_case_submissions_facet(domain, username=None, limit=100):
...
-                },
-                "facet_filter": {
-                    "and": [
-                        {
-                            "term": {
-                                "domain.exact": domain
-                            }
+        "aggs": {
+            "agg_filters": {
+                "filter": es_filter,
```
Comment (snopoke) [inline custom/_legacy/pact/utils.py]: why not put the filter in the main query body?

**Review by snopoke — APPROVED** [review]
Just one question.

**Comment by snopoke** [comment]
Test failure real:
```
ERROR: custom.bihar.tests.test_due_list:TestDueList.test_due_list_by_task_name
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/bihar/tests/test_due_list.py", line 51, in test_due_list_by_task_name
    due_list = list(get_due_list_by_task_name(datetime.utcnow(), case_es=es))
  File "/mnt/commcare-hq-ro/custom/bihar/reports/due_list.py", line 264, in get_due_list_by_task_name
    return ((facet['term'], facet['count']) for facet in es_result['aggs']['agg_filter'][agg_name]['terms'])
KeyError: 'agg_filter'
```

**Comment by emord** [comment]
rebased and removed bihar changes.also changed the query to be in the main query

**Pull request closed without merging**